packages/iov-lisk/src/liskclient.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { Address, BcpAccountQuery, SendTx, TokenTicker, TransactionKind } from "@iov/bcp-types";
+import { Encoding } from "@iov/encoding";
+import { ChainId } from "@iov/tendermint-types";
+
+import { generateNonce, LiskClient } from "./liskclient";
+import { liskCodec } from "./liskcodec";
+import { LiskKeyringEntry } from "./liskkeyringentry";
+
+function pendingWithoutLongRunning(): void {
+  if (!process.env.LONG_RUNNING_ENABLED) {
+    pending("Set LONG_RUNNING_ENABLED to enable long running tests");
+  }
+}
+
+describe("LiskClient", () => {
+  const base = "https://testnet.lisk.io";
+
+  it("can be constructed", () => {
+    const client = new LiskClient(base);
+    expect(client).toBeTruthy();
+  });
+
+  it("can disconnect", () => {
+    const client = new LiskClient(base);
+    expect(() => client.disconnect()).not.toThrow();
+  });
+
+  it("can get chain ID", async () => {
+    const client = new LiskClient(base);
+    const chainId = await client.chainId();
+    expect(chainId).toEqual("da3ed6a45429278bac2666961289ca17ad86595d33b31037615d4b8e8f158bba");
+  });
+
+  it("can get height", async () => {
+    const client = new LiskClient(base);
+    const height = await client.height();
+    expect(height).toBeGreaterThan(6000000);
+    expect(height).toBeLessThan(8000000);
```

Comment:
and in some months this will fail.... and we will have to update.
but fair enough, real testnets

Reply:
Yeah, something like 10 months to go